src/test/kotlin/org/opensearch/commons/alerting/action/DeleteCommentRequestTests.kt

```diff
@@ -0,0 +1,22 @@
+package org.opensearch.commons.alerting.action
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Test
+import org.opensearch.common.io.stream.BytesStreamOutput
+import org.opensearch.core.common.io.stream.StreamInput
+
+class DeleteCommentRequestTests {
+    @Test
+    fun `test delete comment request`() {
```

Comment:
Nitpick: Could you describe the test more in test name? I.e., is this a happy case, etc.